data/Templates/Hl7v2/ADT_A01.liquid

```diff
@@ -80,16 +80,24 @@
 
         {% for obxSegment in obxSegmentLists.OBX -%}
             {% evaluate organizationId_OBX_23 using 'ID/Organization' XON: obxSegment.23 -%}
-            {% include 'Resource/Organization' OBX: obxSegment, ID: organizationId_OBX_23 -%}
+            {% if obxSegment.24 or obxSegment.25 -%}
```

Comment:
Can we put the checks in resources? It might a bit confusing to check the conditions in main templates as OBX.24 and OBX.25 are never mentioned in main templates.

Reply:
It's mainly to avert such a scenario:
When the resource is included in another context without OBX segments, if there is a condition check about OBX inside the resource, the resource won't be rendered.

Take `Patient` for example, it will be included with `PID` and `NK1` separately, it's not appropriate to add either `PID` condition check or `NK1` condition check.